setup.py

```diff
@@ -105,7 +105,7 @@
 # https://groups.google.com/forum/#!topic/astropy-dev/urYO8ckB2uM
 
 # pretty much needed
-install_requires = ['astropy>=0.3', 'numpy>=1.7', 'qtpy>=1.1',
```

Comment:
@pllim, can we treat this change as a separate PR?  In general, I am not opposed, but there are some considerations which are unrelated to this PR and I'd rather treat it as a separate issue.